src/SFA.DAS.Support.Indexer.ApplicationServices.UnitTests/IndexResourceProcessorBase.cs

```diff
@@ -31,15 +30,17 @@ public class IndexResourceProcessorBase
 
         protected const string _indexName = "new_index_name";
         protected Uri _baseUrl;
+        protected string _resourceIdentifier;
+
         protected const int _indexToRetain = 2;
         protected IEnumerable<AccountSearchModel> _accountModels;
         protected SiteResource _accountSiteResource;
         protected SiteResource _userSiteResource;
 
-
         protected void Initialise()
         {
             _baseUrl = new Uri("http://localhost");
+            _resourceIdentifier = "https://citizenazuresfabisgov.onmicrosoft.com/das-at-test-as-ar";
```

Comment:
don't think this real identifier should be hardcoded into the source code

Reply:
changed